package.json

```diff
@@ -1,132 +1,237 @@
-    "moment": "^2.29.4",
-    "papaparse": "^5.2.0",
-    "pico-ui": "^0.0.13",
-    "popper.js": "^1.16.1",
-    "prismjs": "^1.27.0",
-    "prop-types": "^15.7.2",
-    "raw.macro": "^0.4.2",
-    "react": "^16.12.0",
-    "react-ace": "^8.1.0",
-    "react-beautiful-dnd": "^13.0.0",
-    "react-bootstrap": "^2.7.4",
-    "react-dates": "^20.3.0",
-    "react-device-detect": "^1.11.14",
-    "react-dom": "^16.12.0",
-    "react-draggable": "^3.3.2",
-    "react-dropzone": "^10.1.5",
-    "react-file-reader": "^1.1.4",
-    "react-image-magnifiers": "^1.4.0",
-    "react-meta-tags": "^0.7.4",
-    "react-overflow-wrapper": "^0.4.3",
-    "react-redux": "^8.1.1",
-    "react-router-dom": "^6.14.1",
-    "react-scripts": "4.0.0",
-    "react-semantic-ui-range": "^0.7.1",
-    "react-speech-kit": "^3.0.1",
-    "react-tabs": "^4.2.1",
-    "react-virtualized": "^9.22.3",
-    "react-with-direction": "^1.3.1",
-    "reduce": "^1.0.2",
-    "redux-logger": "^3.0.6",
-    "rsuite": "5.24.1",
-    "semantic-ui-react": "^2.1.5",
-    "showdown": "^1.9.1",
-    "showdown-katex": "^0.8.0",
-    "uuid": "^9.0.0",
-    "winston": "^3.3.3"
-  },
-  "devDependencies": {
-    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "@testing-library/jest-dom": "^6.4.5",
-    "@testing-library/react": "12.1.5",
-    "@testing-library/user-event": "14.5.2",
-    "babel-jest": "^26.6.0",
-    "cross-env": "^7.0.2",
-    "eslint-config-airbnb": "^18.1.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.20.0",
-    "husky": "^4.2.5",
-    "lint-staged": "^10.2.4",
-    "prettier": "^2.0.5",
-    "pretty-quick": "^2.0.1",
-    "react-error-overlay": "6.0.9",
-    "sass": "^1.63.6",
-    "update-browserslist-db": "^1.0.13"
-  },
-  "resolutions": {
-    "react-error-overlay": "6.0.9"
-  },
-  "jest": {
-    "moduleNameMapper": {
-      "^axios$": "axios/dist/node/axios.cjs"
-    }
-  }
-}
+{
+  "name": "class-transcribe-frontend",
+  "version": "0.1.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://github.com/classtranscribe/FrontEnd.git"
+  },
+  "scripts": {
+    "start": "node scripts/start.js",
+    "build": "node scripts/build.js",
+    "test": "node scripts/test.js",
+    "predeploy": "npm run build",
+    "lint": "cross-env NODE_ENV=production eslint .",
+    "lint:quiet": "yarn lint --quiet",
+    "lint:fix": "yarn lint --fix",
+    "lint:fix:quiet": "yarn lint:fix --quiet",
+    "prettier": "prettier --list-different \"**/*.{js,jsx,ts,tsx}\"",
+    "prettier:fix": "prettier --write \"**/*.{js,jsx,ts,tsx}\"",
+    "postinstall": "node getDefaultConfig.js",
+    "backend": "node changeBackend.js"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all",
+      "not op_mob >= 1"
```

Comment:
https://github.com/babel/babel/issues/16171#issuecomment-2015227043 - fixing the opera browser issue